src/plugins/validate-semantic/validators/2and3/schemas.js

```diff
@@ -68,3 +68,31 @@ export const validate2And3TypesInDefaultValuesMatchesWithEnum = () => (system) =
       }, [])
     })
 }
+
+export const validate2And3SchemasDefaultsMatchAnEnum = () => (system) => {
+  return system.validateSelectors
+    .allSchemas()
+    .then(nodes => {
+      return nodes.reduce((acc, node) => {
+        const element = node.node || {}
+        let elementEnum, elementDefault, internalLocation
+
+        if(!element || element.enum === undefined || element.default === undefined) {
+          // nothing to do
+          return acc
+        }
+        elementEnum = element.enum
+        elementDefault = element.default
```

Comment:
The `elementEnum` and `elementDefault` variables are used only once. You can either

a) remove them and change the 2nd `if` to
```js
        if(element.enum.indexOf(element.default) === -1) {
```

or

b) assign them earlier and change the 1st `if` to
```js
        if(!element || elementEnum === undefined || elementDefault === undefined) {
```

Reply:
Done, took the first option.